File: src/main/java/br/com/sergioluigi/personalfinancialcontrol/PersonalFinancialControlApplication.java
package br.com.sergioluigi.personalfinancialcontrol;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@EnableJpaAuditing
@EnableJpaRepositories
@SpringBootApplication
public class PersonalFinancialControlApplication {
    public static void main(String[] args) {
        SpringApplication.run(PersonalFinancialControlApplication.class, args);
    }
}

File: src/main/kotlin/com/sergioluigi/personalfinancialcontrol/PersonalFinancialControlApplication.kt
package com.sergioluigi.personalfinancialcontrol

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@EnableJpaAuditing
@EnableJpaRepositories
@SpringBootApplication
class PersonalFinancialControlApplication

fun main(args: Array<String>) {
	runApplication<PersonalFinancialControlApplication>(*args)
}






